File: app/src/main/java/com/example/saka/myapplication/DBManager.java
package com.example.saka.myapplication;

import android.content.ContentValues;
import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

import com.example.saka.myapplication.model.User;

/**
 * Created by Saka on 07-Jun-17.
 */

public class DBManager extends SQLiteOpenHelper {

    public DBManager(Context context) {
        super(context, "birdy.db", null, 1);
    }

    @Override
    public void onCreate(SQLiteDatabase db) {
        String sql = "create table tbUser (_id int primary key autoincrement, _name text," +
                " _email text, uid text)";
        db.execSQL(sql);
    }

    @Override
    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {

    }

    public void insertUser(User user){
        SQLiteDatabase database = getReadableDatabase();
        ContentValues row = new ContentValues();
        row.put("_uid", user.getUid());
        row.put("_name", user.getName());
        row.put("_email", user.getEmail());
    }
}

File: app/src/main/java/com/blockdev/birdy/db/DBManager.kt
package com.blockdev.birdy.db

import android.content.ContentValues
import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper
import com.blockdev.birdy.model.User

/**
 * Created by Saka on 07-Jun-17.
 */
class DBManager(context: Context?) : SQLiteOpenHelper(context, "birdy.db", null, 1) {

    override fun onCreate(db: SQLiteDatabase) {
        val sql = "create table tbUser (_id int primary key autoincrement, _name text," +
            " _email text, uid text)"
        db.execSQL(sql)
    }

    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {
    }

    fun insertUser(user: User) {
        val database = readableDatabase
        val row = ContentValues()
        row.put("_uid", user.uid)
        row.put("_name", user.name)
        row.put("_email", user.email)
    }
}
